### DISCUSSION TITLE
Adult Swim Request

### DISCUSSION THREAD

**Discussion by Jeramy-Babb** [iptv-org/community#1028] [Q&A]
I apologize if this isn't the proper forum, but I have seen request for channels that viewers would like added to the repository so I am here requesting to add Adult Swim.  I'm not sure but I know that adult Swim on the internet is broke up into several channels, with each show   having it's own.  Call me immature for watching cartoons but it's all there is since they removed the adult content. 😄😄😄😄

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
